### ISSUE TITLE
The visualizer tool doesn't work - Visual Studio 2019

### ISSUE THREAD

**Issue by shargon** [zspitz/ANTLR4ParseTreeVisualizer#52]
![image](https://user-images.githubusercontent.com/3167973/74476928-a6ea4800-4eaa-11ea-96bc-39c7472f39e7.png)

With Visual Studio 2019

**Comment by zspitz** [comment]
The error message should probably be more clear, but all it's saying is if there are any parsers or lexers in these assemblies, the visualizer won't be able to use them. Once you press OK, you should be able to use the visualizer as normal.

**Comment by shargon** [comment]
Yes, it works after click OK, but it's not possible to remove it?

**Comment by zspitz** [comment]
I'm currently using this to display the list of available lexers and parsers in the settings. It's important to let the user know that the lexer or parser they want to use is unavailable to the visualizer because the assembly cannot be loaded.

I suppose I could add assemblies for which this error should be ignored, and the message would only appear once. Would that be a good compromise?

Do you have any other thoughts on how to improve this UX?
